addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -1228,6 +1324,37 @@ const actions = {
             args: [channel.uuid, false]
         });
     },
+    /**
+     * @param {Object} param0
+     * @param {function} param0.dispatch
+     * @param {Object} param0.getters
+     * @param {Object} param0.state
+     * @param chatterLocalId
+     * @param {Object} param1
```

Comment:
`param2` (`param1` is `chatterLocalId`)
same remark for some other newly introduced actions